app/views/govuk_publishing_components/components/_metadata.html.erb

```diff
@@ -0,0 +1,62 @@
+<%
+  from ||= []
+  from = Array(from)
+
+  part_of ||= []
+  part_of = Array(part_of)
+
+  other ||= nil
+
+  direction_class = ""
+  direction_class = " direction-#{direction}" if local_assigns.include?(:direction)
+%>
+<div class="gem-c-metadata<%= direction_class %>" data-module="toggle">
```

Comment:
We've got a copy of the toggle module in the components gem now, just replace `toggle` here with `gem-toggle`.